### ISSUE TITLE
POISE should halt if rotational frequency is close to pericenter angular frequency

### ISSUE THREAD

**Issue by RoryBarnes** [VirtualPlanetaryLaboratory/vplanet#142]
A halt needs to be added to ensure POISE is not used at high eccentricity.

**Comment by vmk8826** [comment]
can you please elaborate

**Comment by RoryBarnes** [comment]
Yes, of course! When a planet's instantaneous frequency becomes close to the rotational frequency, i.e. periastron of a high eccentricity orbit, then the assumptions underpinning POISE fail and the code should halt. This feature would be valuable when testing extreme climate variations with POISE coupled to an orbital model and the user was not sure how large the eccentricity could reach (see examples/ChaoticResonances). Whlie there is a dHaltMaxEcc option, the default value is 1.0, but for POISE the default should be something like 0.9. To make this change, one would need to modify halt.c:fniHaltMaxEcc to check if POISE was selected and if true, update dHaltMaxEcc to 0.9.

**Comment by RoryBarnes** [comment]
Actually, it'd probably be better to modify VerifyHalts in halt.c such that if POISE is selected, then dHaltMaxEcc is set to 0.9. Then there will be no need to perform an extra check in fniHaltMaxEcc, which would increase performance since that function is called every timestep.
